VERSION

```diff
@@ -1 +1,14 @@
-LASAGNE-Core version 1.4.3
\ No newline at end of file
+LASAGNE-Core version 1.4.3
+==========================
+
+Compilers:
+---------------
+Visual Studio 2015
+Visual Studio 2017
+g++ 4.8.1
+g++ 6.2.0
+
+Dependencies:
+------------
+ACE-TAO version 6.4.3
+OpenDDS version 3.10
```

Comment:
But than you should add a remark that OpenDDS support isn't supported with Visual Studio 2017 because 3.10 doesn't support that